src/vue/components/morph-parts/morph-data.vue

```diff
@@ -46,19 +46,36 @@
         featuresList: {
           pofs: [ 'grmCase', 'gender', 'part' ],
           others: [
+            { name: 'radical', decorator: 'brackets' },
             { name: 'kind', decorator: 'parenthesize' },
             { name: 'declension', decorator: 'appendtype' },
             { name: 'conjugation', decorator: 'appendtype' },
-            { name: 'note', decorator: 'brackets' }
+            { name: 'note', decorator: 'brackets' },
           ]
         }
       }
     },
     created: function () {
       this.types = Feature.types
     },
-    computed: { 
+    computed: {
+      hasMorhData () {

```

Comment:
minor request, can we fix the typo in this function name? hasMorhData should be hasMorphData. Thanks!

Reply:
Fixed typo!